### PR TITLE
docs(general): 1.3 GSG and API tutorial

### PR DESCRIPTION
**Pull request by bwendlandt-intel**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

- Rework API Tutorial for Auth and refactored APIs
- Update GSG flow with Kong
- Bump branches in clone cmds to v1.3.0

### CODE REVIEW

File: docs/General/loginToRPS.md [hunk 1]
```diff
@@ -45,11 +25,12 @@ The web portal is available for login after the deployment of the MPS, RPS, and
     | **Username**| standalone |
     | **Password**| G@ppm0ym |
 
-3. The home page should look like the following.
+4. The home page should look like the following.
```
Comment (kimberlyxdavis) [inline docs/General/loginToRPS.md:28]: The home page is show below in Figure 1.

Comment (bwendlandt-intel) [reply docs/General/loginToRPS.md]: Updated.

File: docs/Topics/guids.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Each Intel® AMT device has a Global Unique Identifier (GUID) assigned to it by
 There are a number of ways to obtain the GUID on the Intel® AMT device:
 
 - Sample Web UI of the Open AMT Cloud Toolkit 
-- [ConnectedDevices API Method](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.2.0#/Admin/post_admin)
+- [Devices API Method](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.3.0#/Devices/get_devices)
```
Comment (rsdmike) [inline docs/Topics/guids.md:9]:
```suggestion
- [Devices API Method](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.3.0#/Devices/get_api_v1_devices)
```

File: docs/Topics/powerstates.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 --8<-- "References/abbreviations.md"
 
-Actions are specified by number. Use the [PowerCapabilities](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.2.0#/AMT/post_amt) method to return the actions available for a specific device. Use the [PowerState](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.2.0#/AMT/post_amt) method to obtain the current power state.
+Actions are specified by number. Use the [PowerCapabilities](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.3.0#/AMT/get_amt_power_capabilities__guid_) method to return the actions available for a specific device. Use the [PowerState](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.3.0#/AMT/get_amt_power_state__guid_) method to obtain the current power state.
```
Comment (rsdmike) [inline docs/Topics/powerstates.md:3]:
```suggestion
Actions are specified by number. Use the [PowerCapabilities](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.3.0#/AMT/get_api_v1_amt_power_capabilities__guid_) method to return the actions available for a specific device. Use the [PowerState](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.3.0#/AMT/get_api_v1_amt_power_state__guid_) method to obtain the current power state.
```

File: docs/Tutorials/apiTutorial.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
 --8<-- "References/abbreviations.md"
 
-This tutorial demonstrates how to construct an Admin method API call for [*ConnectedDevices*](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.2.0#/Admin/post_admin) using Node.js. The ConnectedDevices method will retrieve information about connected devices, including device GUIDs.
+This tutorial demonstrates how to generate a JWT token for Authorization and construct a API call for [Getting Devices](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.3.0#/Devices/get_devices) using Node.js. This method will retrieve information about all devices, including device GUIDs.
```
Comment (rsdmike) [inline docs/Tutorials/apiTutorial.md:3]:
```suggestion
This tutorial demonstrates how to generate a JWT token for Authorization and construct a API call for [Getting Devices](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.3.0#/Devices/get_api_v1_devices) using Node.js. This method will retrieve information about all devices, including device GUIDs.
```

**Review by rsdmike — CHANGES_REQUESTED** [review]

**Review by rsdmike — APPROVED** [review]

**Pull request merged**